__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const app = require("../app.js");
 const seed = require("../db/seeds/seed.js");
 const data = require("../db/data/test-data");
 const db = require("../db/connection.js");
+const { expect } = require("@jest/globals");
 

```

Comment:
don't need this

Reply:
Yeah I saw that and removed it. Keeps coming up if I don't tab for describe, test or expect in the test area.